operator/go.mod

```diff
@@ -15,3 +15,64 @@ require (
 	k8s.io/client-go v0.23.4
 	sigs.k8s.io/controller-runtime v0.11.1
 )
+
+require (
```

Comment:
do we really require 2 require sections?

Reply:
Yes, these are auto-generated, one is for direct dependencies and other is for indirect dependencies.